content/articles/renewing-lets-encrypt-ssl-certificate.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Renewing a Let's Encrypt SSL Certificate
+excerpt: Instructions to renew a Let's Encrypt SSL certificate for a domain with DNSimple.
+categories:
+- SSL Certificates
+---
+
+<note>
+There are a few important things you should note about [renewing an SSL certificate](/articles/renewing-ssl-certificate) before continuing with this document.
+</note>
+
+All Let's Encrypt SSL certificates, including renewals, are valid for no more than 90 days from their issue date. Thirty days before the certificate expires you will begin receiving renewal notices. If you have not selected the [auto-renewal option](/articles/letsencrypt#auto-renewal), these are the steps to manually renew your Let's Encrypt certificate:
+
+1. Order the SSL certificate renewal
```

Comment:
As I've changed in d40716fdd6c24651f917475f437f15bdb8557c8b, I'd remove the steps in the specific articles, leaving only them in the generic one.